pyCMR/Result.py

```diff
@@ -17,31 +46,46 @@ class Result(dict):
     """
     _location = None
 
-    def download(self, destpath="."):
+    def download(self, destpath=".", unm=None, pwd=None):
         """
         Download the dataset into file system
         :param destPath: use the current directory as default
+        :param unm: earthexplorer username needed for ftp download
```

Comment:
Sorry for confusion. I think those should be the credentials, specified in cmr.cfg.
I didn't have a chance to check everything better, as needed the code to run ASAP.
In my case I had to download near-real-time MODIS data from the nrt server, which needed authentication. That was the simplest way to pass the username/password.
Please, correct me if I misunderstood.
It would be nice to use same config in all of the project.